.changeset/gentle-pots-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-theme": minor
```

Comment:
If this is a minor change, why not target main?

Reply:
The UX-Team and I made a list of things to change for the next major release, this PR was one of those changes... 
So no real reason for `next` but we also don't really need it in `main` 🤷🏼